portfolio/src/main/java/com/google/sps/data/Keys.java

```diff
@@ -0,0 +1,7 @@
+package com.google.sps.data;
+
+public class Keys {
+    public static final String COMMENT_KEY = "Comment";
```

Comment:
Naming wise, this one's not really a key, is it?  I suggested "key" for the other constant names because they are literally used as lookup keys to return values from the key-value parameter list.

"Comment" is really not a lookup key, right?  It's like... an entity name?  I'm not sure what the language is exactly, but since you provide this value when calling new Entity(), it seems like "entity name" is more what describes it.  So I'd go with something like COMMENT_ENTITY_NAME.  And maybe the same kind of thing for the User entity/table in LoginServlet: USER_ENTITY_NAME.

Reply:
Made the names reflect either entities or properties